osgtest/tests/test_550_condorce.py

```diff
@@ -131,7 +138,13 @@ def test_07_ceview(self):
         try:
             src = core.to_str(urlopen(view_url).read())
         except EnvironmentError as err:
-            self.fail('Could not reach HTCondor-CE View at %s: %s' % (view_url, err))
+            debug_contents = 'Could not reach HTCondor-CE View at %s: %s\n%s\n' % (view_url, err, '=' * 20)
+            debug_file = '/var/log/condor-ce/CEViewLog'
+            try:
+                debug_contents += files.read(debug_file, True)
+            except FileNotFoundError:
```

Comment:
```suggestion
            except EnvironmentError:
```
EnvironmentError is still present in Python 3, and FileNotFoundError is a subclass of it, so this will work on both.

Note that on Python 2, a file not found is an OSError, not an IOError. EnvironmentError is a parent of both.